src/cpp/rtps/transport/UDPTransportInterface.h

```diff
@@ -259,7 +265,19 @@ class UDPTransportInterface : public TransportInterface
             eProsimaUDPSocket& socket,
             const Locator& remote_locator,
             bool only_multicast_purpose,
+            bool whitelisted,
             const std::chrono::microseconds& timeout);
+
+    /**
+     * @brief Remove already opened network interfaces
+     *
+     * @param sender_resource_list List of SenderResources already registered in the transport
+     * @param locNames List of available network interfaces
+     */
+    void remove_known_network_interfaces(
```

Comment:
Update doxydoc (including missing param `return_loopback`), the purpose is `Return list of not yet open network interfaces`